crates/sui-single-node-benchmark/src/benchmark_context.rs

```diff
@@ -132,6 +131,63 @@ impl BenchmarkContext {
         root_objects
     }
 
+    pub(crate) async fn prepare_shared_objects(
+        &mut self,
+        move_package: ObjectID,
+        num_shared_objects: usize,
```

Comment:
i notice you have a configurable number of shared objects here, but the executor ignores this and executes all transactions serially, as if there was only one shared object. this is fine for now but it might warrant some comments